ngraph/frontend/tensorflow/src/frontend.cpp

```diff
@@ -12,20 +12,45 @@
 #include "op_table.hpp"
 #include "tf_framework_node.hpp"
 
-using namespace google;
-
-using namespace ngraph::frontend;
-using namespace ngraph::frontend::tf;
+using namespace ::ngraph::frontend;
+using namespace ::ngraph::frontend::tf;
 using namespace ::tensorflow::ngraph_bridge;
 
-using ::tensorflow::GraphDef;
-
 namespace {
-void TranslateGraph(const std::shared_ptr<ngraph::frontend::InputModelTF>& model,
-                    const std::string model_name,
-                    bool fail_fast,
-                    bool no_conversion,
-                    std::shared_ptr<ngraph::Function>& ng_function) {
+void TranslateFWNode(const std::shared_ptr<TFFrameworkNode>& node) {
+    auto type = node->get_op_type();
+
+    const auto TRANSLATE_OP_MAP = get_supported_ops();
+    auto translator_it = TRANSLATE_OP_MAP.find(type);
+    FRONT_END_OP_CONVERSION_CHECK(translator_it != TRANSLATE_OP_MAP.end(), "No translator found for ", type, " node.");
+
+    ngraph::OutputVector ng_inputs;
+    NamedInputs named_inputs;
+    size_t input_port_idx = 0;
+    for (auto& input : node->inputs()) {
+        ng_inputs.push_back(input.get_source_output());
+        named_inputs[input_port_idx++] = {input.get_source_output()};
+    }
+
+    NodeContext node_ctx(*node->get_decoder(), named_inputs);
+    auto new_node_outputs = translator_it->second(node_ctx);
+    SetTracingInfo(node_ctx.get_name(), new_node_outputs.front());
+
+    auto new_output = new_node_outputs.begin();
+    auto old_outputs = node->outputs();
+    auto old_output = old_outputs.begin();
+
+    for (; new_output != new_node_outputs.end() && old_output != old_outputs.end(); ++old_output, ++new_output) {
```

Comment:
we expect that `new_node_outputs.size() == old_outputs.size()`?
if so, it's better to add check here

Reply:
We expect that new_node_output.size() >= old_outputs.size(). I think.

Reply:
I left this logic unchanged. I think Maxim to be aware of it better.